spec/iss67-spec.js

```diff
@@ -10,7 +10,9 @@ import {init as generateConfig} from '../commands/generate';
 
 describe('editorconfig', () => {
 	beforeEach(() => {
-		waitsForPromise(() => atom.packages.activatePackage('editorconfig'));
+		waitsForPromise(() => {
+			return atom.packages.activatePackage('editorconfig');
+		});
```

Comment:
Really? i thought the return statement would be absolutely necessary. I'll rewind these kinds of lines.